### PR TITLE
Expose some methods

### PR DESCRIPTION
**Pull request by mrvoorhe**
Expose some methods that the unity linker needs to call or override

### CODE REVIEW

File: linker/Mono.Linker/Driver.cs [hunk 1]
```diff
@@ -172,7 +172,7 @@ void Run ()
 			p.Process (context);
 		}
 
-		static void AddCustomStep (Pipeline pipeline, string arg)
+		protected void AddCustomStep (Pipeline pipeline, string arg)
```
Comment (marek-safar) [inline linker/Mono.Linker/Driver.cs]: Why did you remove static?

File: linker/Mono.Linker/Driver.cs [hunk 2]
```diff
@@ -233,7 +233,7 @@ static IStep ResolveStep (string type)
 			return (string []) lines.ToArray (typeof (string));
 		}
 
-		static I18nAssemblies ParseI18n (string str)
+		protected I18nAssemblies ParseI18n (string str)
```
Comment (marek-safar) [inline linker/Mono.Linker/Driver.cs]: Why did you remove static?

File: linker/Mono.Linker/Driver.cs (hunk 1 above)
Comment (mrvoorhe) [inline linker/Mono.Linker/Driver.cs]: Side effect of some other changes.  Looks like I can shift some things around and change this back to static.

I will add back static

**Comment by mrvoorhe** [comment]
@marek-safar Fixed.  static added back to both methods
